snuba/query/processors/physical/__init__.py

```diff
@@ -41,6 +41,10 @@ def process_query(self, query: Query, query_settings: QuerySettings) -> None:
     def get_from_name(cls, name: str) -> "ClickhouseQueryProcessor":
         return cast("ClickhouseQueryProcessor", cls.class_from_name(name))
 
+    @classmethod
+    def is_valid_delete_query(self, query: Query) -> bool:
+        return True
```

Comment:
we don't want a @classmethod here since the method depends on an instance of the class, not the class itself. I think actually it might be better to have this kind of check on the `Query` itself, something like `is_delete` but I'm not sure, for now I'd actually leave it out of this PR